docs/api-reference/cli.md

```diff
@@ -219,14 +219,17 @@ $ export QUILT_TRANSFER_MAX_CONCURRENCY=20
 ```
 
 ### `XDG_*`
-Quilt uses platformdirs for Python to determine where to write data. You can therefore
-override the following path constants with environment variables using the XDG
```

Comment:
It doesn't make sense to say "override the following path constants with...." and then not list the path constants until the very end of the section (i.e. after an un-related example section). Move the "Constants" section immediately below this initial sentence, and then have the lambda example code in an "Example" section beneath the "Constants" section.

Reply:
We never say "path constants" as that was in the old version. One of these .md files is generated so only one reviewer comment is needed per mistake.

"following" is misleading, I'll clean that up.